conf/defaults.ini

```diff
@@ -717,13 +717,19 @@ global_alert_rule = -1
 
 #################################### Unified Alerting ####################
 [unified_alerting]
+# Disable unified alerting
+enabled = true
+
+# Set the organisations (IDs) for which to disable unified alerting, separated by commas. It has no effect if unified alerting in disabled.
+disabled_orgs = 
+
 # Specify the frequency of polling for admin config changes.
 admin_config_poll_interval_seconds = 60
 
 #################################### Alerting ############################
 [alerting]
-# Disable alerting engine & UI features
-enabled = true
+# Enable legacy alerting engine & UI features
+enabled = false
```

Comment:
is there some validation that both unified and legacy alerting are not enabled at the same time?

Reply:
There legacy alerting engine [is disabled if the new is enabled](https://github.com/grafana/grafana/blob/8feeaab99630efa5b359a383bd0e3e561a718e9f/pkg/services/alerting/engine.go#L48) but we don't inform the users about it.
So it's good to return an error if both are enabled.